dev/sparktestsupport/modules.py

```diff
@@ -29,7 +29,7 @@ class Module(object):
     changed.
     """
 
-    def __init__(self, name, dependencies, source_file_regexes, build_profile_flags=(),
+    def __init__(self, name, dependencies, source_file_regexes, build_profile_flags=(), environs={},
```

Comment:
Minor naming nit: I think `environ` would be _slightly_ more idiomatic, given that Python calls the dictionary itself `os.environ`.